src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java

```diff
@@ -699,6 +687,34 @@ private void scrape(List<MetricFamilySamples> mfs) {
       samples.add(new MetricFamilySamples.Sample(
           "cloudwatch_exporter_scrape_error", new ArrayList<>(), new ArrayList<>(), error));
       mfs.add(new MetricFamilySamples("cloudwatch_exporter_scrape_error", Type.GAUGE, "Non-zero if this scrape failed.", samples));
+
+      String buildVersion = "";
+      String releaseDate = "";
+      int errorFlag = 0;
+      try {
+          final Properties properties = new Properties();
+          properties.load(CloudWatchCollector.class.getClassLoader().getResourceAsStream(".properties"));
+          buildVersion = properties.getProperty("BuildVersion");
+          releaseDate = properties.getProperty("ReleaseDate");
+
+      }
+      catch (IOException e) {
+          buildVersion = "Error";
+          releaseDate = "Error";
+          errorFlag = 1;
+          LOGGER.log(Level.WARNING, "CloudWatch build info scrape failed", e);
+      }
+
+      labelNames.add("build_version");
+      labelValues.add(buildVersion);
+      labelNames.add("release_date");
+      labelValues.add(releaseDate);
+
+      samples = new ArrayList<>();
+      samples.add(new MetricFamilySamples.Sample(
+          "cloudwatch_exporter_build_info", labelNames, labelValues, errorFlag));
+      mfs.add(new MetricFamilySamples("cloudwatch_exporter_build_info", Type.GAUGE, "Non-zero if build info scrape failed.", samples));
```

Comment:
the build info metric should always have value `1`. If fetching the build info fails, make the label values `unknown` to match how other build info metrics behave.